rebasehelper/specfile.py

```diff
@@ -160,6 +170,22 @@ def __str__(self):
                 content.append(line + '\n')
         return ''.join(content)
 
+    @classmethod
+    def get_comment_span(cls, line: str, section: str) -> Tuple[int, int]:
+        """Gets span of a comment depending on the section.
+
+        Args:
+            line: Line to find the comment in.
+            section: Section the line is in.
+
+        Returns:
+            Span of the comment
+
```

Comment:
I think the docstring should explain what (and why) is returned when no comment is found.

Reply:
Just a nitpick, I would replace
`to length of line to make it convenient to use the result in a slice`
with
`to the length of the line for convenient use in a slice`.